backend/src/exercise/websocket-handler/join-exercise-handler.ts

```diff
@@ -10,9 +10,15 @@ export const registerJoinExerciseHandler = (
         client,
         'joinExercise',
         (exerciseId: string, clientName: string, callback): void => {
-            const clientId = clientMap
-                .get(client)
-                ?.joinExercise(exerciseId, clientName);
+            const clientWrapper = clientMap.get(client)!;
```

Comment:
The `client` is always already in the `clientMap` when this function is called? Maybe add a comment for this.